sn_cli/src/subcommands/files.rs

```diff
@@ -137,11 +137,49 @@ pub(crate) async fn files_cmds(
                 );
             }
 
-            let download_dir = dirs_next::download_dir().unwrap_or(root_dir.to_path_buf());
+            let mut download_dir = root_dir.to_path_buf();
+            let mut download_file_name = file_name.clone();
+            if file_name.is_some() {
+                // file_name may direct the downloaded data to:
```

Comment:
```suggestion
            if let Some(file_name) = file_name {
                // file_name may direct the downloaded data to:
```
With this we don't need the `.expect("...")` below